internal/pipeline/executor.go

```diff
@@ -120,16 +121,14 @@ func (p *pkg) Pipeline() *Pipeline {
 // and returns list of package paths that this pkg depends on.
 // This is one of the critical pieces of code
 func (p *pkg) resolveSources() ([]string, error) {
-	pipeline := p.Pipeline()
-
 	var pkgPaths []string
-	for _, s := range pipeline.Sources {
-		paths, err := resolveSource(s, p.Path())
-		if err != nil {
-			return nil, err
-		}
-		pkgPaths = append(pkgPaths, paths...)
+	// TODO: since we are considering only support './*', we can simplify
+	// this part
```

Comment:
Let's not try to simplify further because we will learn more about `sources` as we go along, so this is good for now.